packages/cli/lib/lib/webpack/create-load-manifest.js

```diff
@@ -24,18 +28,23 @@ module.exports = assets => {
       weight: 1
     },
   },
-  manifest = {
-    '/': defaults
-  };
+  manifest = {};
 
   let path, css, obj;
   scripts.forEach((filename, idx) => {
     css = styles[idx];
     obj = Object.assign({}, defaults);
     obj[filename] = { type:'script', weight:0.9 };
     if (css) obj[css] = { type:'style', weight:0.9 };
-    path = filename.replace(/route-/, '/').replace(/\.chunk(\.\w+)?\.js$/, '').replace(/\/home/, '/');
-    manifest[path] = obj;
+    path = filename.replace(/route-/, '/')
+      .replace(/\.chunk(\.\w+)?\.js$/, '')
+      .replace(/\.chunk(\.\w+)?\.esm\.js$/, '')
+      .replace(/\/home/, '/');
```

Comment:
We already relied on `home.js` before (which we should probably fix), but by removing:

```js
manifest: {
  '/': defaults
}
```

...we're now _requiring_ a file named `routes/home.js` in order to populate a manifest entry for the `/` path.

Reply:
oh nice catch, this was an intermittent thing when i was trying stuff.
I can put back 
```
manifest: {
  '/': defaults
}
```

Will that address your concern?

Reply:
I _think_ so, tho I'm not 100% sure